packages/mobile/src/account/SyncBankAccountScreen.test.tsx

```diff
@@ -2,13 +2,13 @@ import { render, waitFor } from '@testing-library/react-native'
 import * as React from 'react'
 import 'react-native'
 import { Provider } from 'react-redux'
+import { Actions } from 'src/account/actions'
 import SyncBankAccountScreen from 'src/account/SyncBankAccountScreen'
-import { createMockStore, getMockStackScreenProps } from 'test/utils'
-import { mockAccount, mockPrivateDEK } from 'test/values'
 import { createFinclusiveBankAccount, exchangePlaidAccessToken } from 'src/in-house-liquidity'
+import { navigate, replace } from 'src/navigator/NavigationService'
```

Comment:
Totally orthogonal to this PR, but do we automatically sort imports? Might be nice to do (I believe eslint has a rule for this); would be a bit cleaner and reduce unnecessary diffs.